src/_guides/language/language-tour.md

```diff
@@ -1559,9 +1559,9 @@ assert(foo() == null);
 
 ## Operators
 
-Dart defines the operators shown in the following table.
-You can override many of these operators, as described in
-[Overridable operators](#overridable-operators).
+Dart supports the operators shown in the following table.
+You can define your own operators, as described in the section on
```

Comment:
@eernstg - Good points. In rereading, I agree that "define your own" is misleading.

How about:

```diff
- You can override many of these operators, 
+ You can implement and override many of these operators, 
```

My concern with the original formulation is that it might give readers who are new to Dart the impression that one can _only_ override operators.

Reply:
Right, we don't want to imply that there must be an override relation, but even 'implement' is slightly dangerous (if it is taken to mean that there must be an `implements` relationship): We keep hinting that an operator cannot be added to an arbitrary interface. It's difficult to keep it precise as well as concise, but here's one more way to say it:

```suggestion
With many of these operators (e.g., `+`), a class can declare an implementation (`operator +`), thus adding that operator to the class or overriding a declaration in a superinterface. This is described in the section on
```

Reply:
Thanks for the suggestion.

> It's difficult to keep it precise as well as concise

Yes. Here's one more try at keeping it concise; how about (additions in bold):

> You can **implement and** override many of these operators **as class members**, ...

Or more simply:

> You can **implement** many of these operators **as class members**, ...

(leaving it understood that operators, like other members, can be overridden).

Trying to find a balance here for the Language Tour, in being sufficiently accurate without getting too much into details. @kwalrath - any further suggestions or comments?

Reply:
I like the 'more simply' variant, and I'm sure @kwalrath has good further suggestions. ;-)

Reply:
I like the "more simply" version, too.

Reply:
Done. Changed the text as follows:

> Dart supports the operators shown in the following table.
You can implement many of these [operators as class members](#_operators).

PTAL @kwalrath